packages/palette_generator/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.3.3
+
+* Avoid dynamic calls in equality checks.
```

Comment:
Nit: Avoids

(We now have a style guide for CHANGELOG entries.)

Reply:
Fixeded